Tasks/JBossDeployer/Tests/L0runsJBossDeployerList.ts

```diff
@@ -0,0 +1,47 @@
+import ma = require('vsts-task-lib/mock-answer');
+import tmrm = require('vsts-task-lib/mock-run');
+import path = require('path');
+import fs = require('fs');
+
+let taskPath = path.join(__dirname, '..', 'jbossdeployer.js');
+let tmr: tmrm.TaskMockRunner = new tmrm.TaskMockRunner(taskPath);
+
+process.env["ENDPOINT_URL_mock_jbossEndpoint"] = "https://example.test/v0.1";
+process.env["ENDPOINT_AUTH_PARAMETER_mock_jbossEndpoint_USERNAME"] = "test_username";
+process.env["ENDPOINT_AUTH_PARAMETER_mock_jbossEndpoint_PASSWORD"] = "test_password";
+
+tmr.setInput('jbossEndpoint', 'mock_jbossEndpoint');
+tmr.setInput('file', 'mock_file');
+tmr.setInput('force', 'true');
+tmr.setInput('disabled', 'true');
+tmr.setInput('name', 'mock_name');
+tmr.setInput('runtimeName', 'mock_runtimeName');
+tmr.setInput('serverMode', 'domain');
+tmr.setInput('options', 'mock_options');
+tmr.setInput('undeploy', 'false');
+tmr.setInput('serverGroups', 'list_groups');
+tmr.setInput('listedGroups', 'mock_list_groups');
```

Comment:
It would be nice if you can also test multiple groups.  They should be comma separated with no spaces in between.